echoregions/formats/regions2d.py

```diff
@@ -277,6 +288,54 @@ def swap_val(val):
         regions.loc[:] = regions.apply(replace_depth, axis=1)
         return regions
 
+    def convert_points(
+        self, points, convert_time=True, convert_depth_edges=True, offset=0, unix=False
+    ):
+        """Convert x and y values of points from the EV format.
+        Returns a copy of points.
+        Parameters
+        ----------
+        points : list, dict
+            point in [x, y] format or list/dict of these
+        convert_time : bool
+            Whether to convert EV time to datetime64, defaults `True`
+        convert_depth_edges : bool
+            Whether to convert -9999.99 edges to real range values.
+            Min and max ranges must be set manually or by calling `set_range_edge_from_raw`
+        offset : float
+            depth offset in meters
+        unix : bool
+            unix : bool
+            Whether or not to output the time in the unix time format
+        Returns
+        -------
+        points : list or dict
+            single converted point or list/dict of converted points depending on input
+        """
+        return self._parser.convert_points(
```

Comment:
I think we should just move `convert_points`  from `evr_parser` to here. Conceptually it is cleaner if the parser just parse points, and for downstream manipulations to be carried out in `Region2D`.